addons/web/static/src/js/views/kanban/kanban_column_progressbar.js

```diff
@@ -92,8 +92,11 @@ var KanbanColumnProgressBar = Widget.extend({
         const subgroupCounts = {};
         for (const key of Object.keys(this.colors)) {
             const subgroupCount = this.columnState.progressBarValues.counts[key] || 0;
-            if (this.activeFilter.value === key && subgroupCount === 0) {
+            if (this.activeFilter.value === key && subgroupCount <= 0) {
```

Comment:
I didn't figure out in which case subgroupCount could be negative ?

Maybe just add a comment to explain why ? :)

Thanks :)

Reply:
Hello @tlibioulle  
if we are considering only case of project module then no need to change sign it works fine.
I’ve changed the sign for module like fleet, crm where we have four filter right, __false: muted which is grey color, according to it’s calculation in _readProgressBarGroup method it will be negative in many case, so when __false count are assigned to subgroupCount, then our subgroupCount will be negative.

see: [https://drive.google.com/file/d/1GU2_QpuKOK4pwGrFWf0eC4FUVqXCxzXT/view](url)

hope this link will be helpful

Thank you:)

Reply:
Oh now I see ! It triggered me so I went one step further and found this :
https://www.awesomescreenshot.com/video/3530274?key=d51b4141ce32f5e72f5f3cc604c7e38a

I bet we expect something like this :
https://www.awesomescreenshot.com/video/3530284?key=0d7f47ef8e7f278c0e74b84680970452

I tried with something like this :
```suggestion
            if (this.activeFilter.value === key && this.columnState.count === 0 && subgroupCount <= 0) {
                this.activeFilter = {};
                this.trigger_up('kanban_load_column_records', {
                    activeFilter: this.activeFilter
                });
            }
            subgroupCounts[key] = this.activeFilter.value === key ? this.columnState.count : subgroupCount;
```

What do you think about it ?